cmd/dosa/schema.go

```diff
@@ -136,6 +151,47 @@ func (c *SchemaCmd) doSchemaOp(name string, f func(dosa.AdminClient, context.Con
 	return nil
 }
 
+func (c *SchemaCmd) doSchemaOpInJavaClient(op string) {
+	cmd := "java"
+	var schemaOp string
+	if strings.Compare(op, schemaCheck) == 0 {
```

Comment:
It shouldn't matter right? Even they use -j,  UPSERT AND CAN_UPSERT should just work as it is. Otherwise users will be confused that how they upsert java entity.